packages/trpc/server/routers/viewer/organizations/adminVerify.handler.ts

```diff
@@ -42,18 +38,13 @@ export const adminVerifyHandler = async ({ input }: AdminVerifyOptions) => {
 
   const acceptedEmailDomain = foundOrg.members[0].user.email.split("@")[1];
 
-  const metaDataParsed = teamMetadataSchema.parse(foundOrg.metadata);
-
-  await prisma.team.update({
+  await prisma.organizationSettings.update({
     where: {
-      id: input.orgId,
+      teamId: input.orgId,
     },
     data: {
-      metadata: {
-        ...metaDataParsed,
-        isOrganizationVerified: true,
-        orgAutoAcceptEmail: acceptedEmailDomain,
-      },
```

Comment:
We no longer need to update metadata for this as it is handled in organizationSettings